pkg/ccl/crosscluster/logical/dead_letter_queue.go

```diff
@@ -129,19 +129,35 @@ func (dlq *loggingDeadLetterQueueClient) Log(
 }
 
 type deadLetterQueueClient struct {
-	ie isql.Executor
+	ie              isql.Executor
+	tableIDToPrefix map[int32]string
 }
 
-func (dlq *deadLetterQueueClient) Create(ctx context.Context, tableIDs []int32) error {
-	if _, err := dlq.ie.Exec(ctx, "create-enum", nil, writeEnumStmt); err != nil {
-		return errors.Wrap(err, "failed to create crdb_replication_mutation_type enum")
-	}
+func (dlq *deadLetterQueueClient) Create(ctx context.Context) error {
+	// Store unique db names to avoid executing identical enum statements.
```

Comment:
why is this necessary, given the create statement contains the `IF NOT EXISTS` clause? If the enum already exists, the statement should noop.